components/style/binding_tools/regen.py

```diff
@@ -542,49 +541,11 @@ def zero_size_type(ty, flags):
         return 1
 
     print("OK")
+    print("(please test with ./mach test-stylo)")
```

Comment:
Can we run this inline? It seems to me it's pretty convenient to have the results of struct layout tests right after generating the bindings.